README.md

```diff
@@ -1,13 +1,17 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The assignment for week 17 of the TECHNIGO Boot Camp was to create a couple of RESTful endpoints which return either an array of data, or a single item. This was our very first brief dedicated to backend.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We started out with some initial hard-coded data, stored as JSON files. I choose to work with an API listing 500 book reviews.
+
+The minimum requirements were to create at least 2 routes using Express in Node, with the API being RESTful. One of the endpoints had to return a collection of results, and one of the endpoints a single result.
+
+My first endpoint lists an array of elements being all books, and also accepts a filter via query parameters to generate books specified by author. My second endpoint returns a single element generated by the unique book id.
+
+Next step will be to expand the routes and include more endpoints, and to build a frontend to go along with the project.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+To view an example of a deployed endpoint, please follow this link: http://small-express-api.herokuapp.com/books
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓